bins/packages/hd-idle/hd-idle.sh

```diff
@@ -9,7 +9,12 @@ dependencies_hd-idle() {
 
 download_hd-idle() {
     echo "[+] downloading and installing hd-idle"
-    go get github.com/adelolmo/hd-idle
+    go get ${HD_IDLE_REPOSITORY}@${HD_IDLE_VERSION}
+}
+
+prepare_hd-idle() {
+    echo "[+] prepare hd-idle"
+    github_name "hd-idle-${HD_IDLE_VERSION}"
```

Comment:
I would recommend to use `${HD_IDLE_VERSION:0:8}` to get a short id (otherwise filename will be large)